site/src/main/scala/org/typelevel/sbt/TypelevelSitePlugin.scala

```diff
@@ -84,6 +98,13 @@ object TypelevelSitePlugin extends AutoPlugin {
       Helium
```

Comment:
@jenshalm I couldn't figure it if/how it's possible to move this config into the `TypelevelHeliumExtensions`, would appreciate any pointers.

Reply:
I took another look at this and I was able to do it :)